apps/kubers/Makefile

```diff
@@ -0,0 +1,14 @@
+APP_NAME = polarsquad/kubers
+
+all: v1
```

Comment:
Maybe rename the ‘v1’ to ‘build’ so it’s more describing and we don’t have multiple versions